cognite/extractorutils/base.py

```diff
@@ -193,13 +195,23 @@ def __enter__(self) -> "Extractor":
         Returns:
             self
         """
+
+        # Environment Variables
+        self.env_file_path = find_dotenv()
+        if len(self.env_file_path) > 0:
```

Comment:
To check if a string is not None and not empty, you can just use it as a boolean directly 🐍
``` suggestion
        if self.env_file_path:
```